binder/requirements.txt

```diff
@@ -1,2 +1,5 @@
 ../scrapbook
```

Comment:
mybinder.org complains about this not being a valid requirement. Should it be `-e../scrapbook`?

Reply:
Hmm nope, tried that too. And a few other options (this is affecting papermill binders too). @willingc said she might be able to help here.

Reply:
Ah! `-e..` as it should be the directory that contains the `setup.py`

Reply:
Nope, that fails too :/

Reply:
Carol mentioned that it might be related to how the docker image sets up such that it's not available.

Reply:
@MSeal I applied your PR on my fork and got it to work on binder. I removed `../scrapbook` on line 1 and added a final requirement `-e .` See https://github.com/willingc/scrapbook/blob/binderAndFixes/binder/requirements.txt#L5